Doc/c-api/exceptions.rst

```diff
@@ -438,17 +438,18 @@ Querying the error indicator
 
 .. c:function:: void PyErr_Fetch(PyObject **ptype, PyObject **pvalue, PyObject **ptraceback)
 
-    As of 3.12, this function is deprecated. Use :c:func:`PyErr_GetRaisedException` instead.
-
    Retrieve the error indicator into three variables whose addresses are passed.
    If the error indicator is not set, set all three variables to ``NULL``.  If it is
    set, it will be cleared and you own a reference to each object retrieved.  The
    value and traceback object may be ``NULL`` even when the type object is not.
 
    .. note::
 
-      This function is normally only used by code that needs to catch exceptions or
-      by code that needs to save and restore the error indicator temporarily, e.g.::
+      This function is normally only used by legacy code that needs to catch
+      exceptions or by code that needs to save and restore the error indicator
+      temporarily.
```

Comment:
```suggestion
      exceptions or save and restore the error indicator temporarily.
```

More concise, avoid repeating "code that needs to" here and ensure "legacy" applies to it too.